### DISCUSSION TITLE
Merge Discussion Groups?

### DISCUSSION THREAD

**Discussion by dbareis** [hluk/CopyQ#2003] [Ideas]
To me it looks like the [Google Discussion Group is deadish](https://groups.google.com/g/copyq), now that I have found this discussion group (and it seems more "alive"), it seems to me the google one could be closed to get more users in a single group.  Maybe the google group could be repurposed for only new release announcements and made read only for everyone else pointing users to these discussions?

**Comment by dbareis** [comment]
At the very least all groups should link to all others.

**Comment by hluk** [comment]
I haven't got much time to reply my mail recently (and even check notifications on GitHub). I usually try to reply faster when I know the answer and it is not very time consuming.

For me, GitHub issues and discussion is a bit better because it is easier to find and refer to previous stuff.
